rules/S6875/java/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "SequencedCollection method should be used to add or remove first or last element",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "Java 21"
```

There's already some rules using `java21 ` for example [S6879/java/metadata.json](https://github.com/SonarSource/rspec/blame/master/rules/S6879/java/metadata.json)
```suggestion
    "java21"
```